src/main/proto/spine/options.proto

```diff
@@ -834,28 +847,39 @@ message PatternOption {
 }
 
 // Specifies the message to show if a validated field happens to be invalid.
-// Is applicable only to messages.
-// Repeated fields are supported.
-//
-// Example: Using the `(if_invalid)` option.
 //
-//     message Holder {
-//         MyMessage field = 1 [(validate) = true,
-//                              (if_invalid).error_msg = "The field is invalid."];
-//    }
+// It is applicable only to message fields marked with `(validate)`.
+// Repeated fields are supported.
 //
 message IfInvalidOption {
 
-    // The default error message for the field.
-    option (default_message) = "The message must have valid properties.";
+    // The default error message.
+    option (default_message) = "The field `{parent.type}.{field.path}` of type `{field.type}` is invalid. The field value: `{field.value}`.";
```

Comment:
Why are we having here `field.path` instead of `field.name`? We refer to the immediate placement of the field in this context.

Reply:
We have the combination of `{parent.type}.{field.name}` above.